schemas/expconf/v0/log-policy.json

```diff
@@ -3,22 +3,32 @@
     "$id": "http://determined.ai/schemas/expconf/v0/log-policy.json",
     "title": "LogPolicy",
     "additionalProperties": false,
-    "required": [
-        "pattern",
-        "action"
```

Comment:
Arguably you should be adding a check that prevents somebody from submitting a log policy that has pattern without either a name or an action.

Legacy policies must have an action, and new policies must have a name.

Reply:
Add a check that requires users to submit either a modern policy or a legacy policy.